src/modm/communication/xpcc/dispatcher.hpp.in

```diff
@@ -48,6 +48,9 @@ namespace xpcc
 		void
 		update();
 
+		void
+		update_once_rx();
```

Comment:
All function names in this header are `camelCase` but this is `snake_case`. That feels a bit inconsistent.

Reply:
Done, CamelCase it is :wink: